### PR TITLE
use throng instead of express-cluster for simplicity

### PR DESCRIPTION
**Pull request by terite**
I'm not sure if worker cycling is worth keeping after PR #15 

@diwu1989 
@BetterWorks/enterprise

### CODE REVIEW

File: routes/circularize.js [hunk 1]
```diff
@@ -24,10 +24,14 @@ router.get('/', function(req, res) {
 
     var type = response.headers['content-type'] || '';
 
-    if (response.statusCode >= 400 || type.indexOf('image') === -1) {
-      res.status(response.statusCode);
-      res.send(body);
-      return;
+    if (response.statusCode >= 400) {
+      res.status(400).send('src returned HTTP ' + res.statusCode);
```
Comment (terite) [inline routes/circularize.js]: I snuck this in here instead of just forwarding unknown responses as our own

File: routes/index.js [hunk 2]
```diff
@@ -3,7 +3,6 @@
 var Canvas = require('canvas');
 var crypto = require('crypto');
 var express = require('express');
-var path = require('path');
```
Comment (terite) [inline routes/index.js]: to fix jshint

**Comment by ayu48** [comment]
i think it lgtm. tested locally.

**Comment by diwu1989** [comment]
lgtm

**Pull request merged**